deb/make-keyring.sh

```diff
@@ -5,14 +5,18 @@ set -eux
 BASE=$(pwd)/brave-keyring
 KEY_URL=${KEY_URL:-https://brave-browser-apt-release.s3.brave.com/brave-core.asc}
 
-if [ ! -f brave-core.asc ]; then
+if [ ! -f ./brave-core.asc ]; then
   echo "No public key file (./brave-core.asc) detected in present working directory. Downloading)"
   wget "$KEY_URL" -O brave-core.asc
 fi
 
 # Uncomment when/if we decide to move the keyring file to /usr/share/keyrings/ as recommended in:
 # https://wiki.debian.org/DebianRepository/UseThirdParty
 
+if [ ! -d "${BASE}/etc/apt/trusted.gpg.d/" ]; then
```

Comment:
Not necessary to fix, but FYI, `mkdir -p` doesn't throw an error when the directory already exists, so the `if` statement is unnecessary.